### PR TITLE
Grammar Update V3

### PR DESCRIPTION
**Pull request by RapidThrower264**
New Stuff
- Adds `UNDEAD` stat and powder symbol (`᠅`)

Changes
- Allow the ability to generate any combination of displayed item, recipe and description using `/gen full`.
- Add more information to `/gen help` for full item generation (and reducing ambiguity).
- You can now use `enchanted` instead of `enchant` to make an item display with the enchantment glint.
- Renamed `Critical` to `Crit` (finally).

Bug Fix
- Fix 1 line item generations having extra space at the bottom.

### CODE REVIEW

File: src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java [hunk 1]
```diff
@@ -118,8 +118,8 @@ public void generateFullItem(GuildSlashEvent event,
         hidden = (hidden != null && hidden);
         event.deferReply(hidden).queue();
 
-        // checking that there is a recipe and/or item to be displayed
-        if (itemName == null && recipe == null) {
+        // checking that there is more two or more different items to merge the images
```
Comment (Aerhhh) [inline src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java:121]:
```suggestion
        // checking that there is two or more different items to merge the images
```

File: src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java [hunk 2]
```diff
@@ -128,15 +128,18 @@ public void generateFullItem(GuildSlashEvent event,
...
         }
 
         // building the item for the which is beside the description
         BufferedImage generatedItem = null;
-        if (itemName != null) {
-           generatedItem = builder.buildUnspecifiedItem(event, itemName, extraModifiers);
+        if (itemID != null) {
+           generatedItem = builder.buildUnspecifiedItem(event, itemID, extraModifiers);
             if (generatedItem == null) {
                 return;
```
Comment (Aerhhh) [inline src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java:144]: Wouldn't this just make the command fail? Should we not send a reply saying something went wrong?

Comment (RapidThrower264) [reply src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java:144]: If someone decides to not provide a item id, then it doesn't render it to the image. But if a full image cannot be rendered, then it says it

File: src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java [hunk 3]
```diff
@@ -118,8 +118,8 @@ public void generateFullItem(GuildSlashEvent event,
         hidden = (hidden != null && hidden);
         event.deferReply(hidden).queue();
 
-        // checking that there is a recipe and/or item to be displayed
-        if (itemName == null && recipe == null) {
+        // checking that there is two or more different items to merge the images
```
Comment (Aerhhh) [inline src/main/java/net/hypixel/nerdbot/command/ItemGenCommands.java:121]:
```suggestion
        // checking that there are two or more different items to merge the images
```

**Pull request merged**